scripts/deploy/github/build-images.sh

```diff
@@ -25,5 +25,5 @@ docker build -t "${REGISTRY}/kfp-tekton/persistenceagent:latest" -f backend/Dock
 docker build -t "${REGISTRY}/kfp-tekton/metadata-writer:latest" -f backend/metadata_writer/Dockerfile . && docker push "${REGISTRY}/kfp-tekton/metadata-writer:latest" &
 docker build -t "${REGISTRY}/kfp-tekton/scheduledworkflow:latest" -f backend/Dockerfile.scheduledworkflow . && docker push "${REGISTRY}/kfp-tekton/scheduledworkflow:latest" &
 docker build -t "${REGISTRY}/kfp-tekton/cache-server:latest" -f backend/Dockerfile.cacheserver . && docker push "${REGISTRY}/kfp-tekton/cache-server:latest" &
-
+docker build -t "${REGISTRY}/kfp-tekton/frontend:latest" -f frontend/Dockerfile . && docker push "${REGISTRY}/kfp-tekton/frontend:latest" &
```

Comment:
just fyi we didn't add frontend in our github actions because it sometimes get over the github action vm quota. We do test our frontend code in our ibm cluster ci by running the `Manual Trigger - e2e` at https://cloud.ibm.com/devops/pipelines/tekton/f85a45d2-36fc-45fb-a449-b01c13cf5191?env_id=ibm%3Ayp%3Aus-south

Reply:
The github action e2e script itself is communicating between the python client to api server, so it never talks to the ui itself.

Reply:
@yhwang fyi maybe we should have another github action to build ui images when the ui code is modified.